server/controllers/project.controller.js

```diff
@@ -30,14 +30,15 @@ export function updateProject(req, res) {
         $set: req.body
       },
       {
-        new: true
+        new: true,
```

Comment:
For `findOneByIdAndUpdate`, you have to explicitly run the mongoose validators. I hate it.